code/modules/integrated_electronics/subtypes/weaponized.dm

```diff
@@ -138,9 +138,9 @@
 	A.preparePixelProjectile(target, src)
 	A.fire()
 	if(ismob(loc.loc))
-		installed_gun.shoot_live_shot(loc.loc)
+		installed_gun.shoot_live_shot(loc.loc, FALSE, null, TRUE, 0)
 	else
-		installed_gun.shoot_live_shot() //Shitcode, but we don't have much of a choice
+		installed_gun.shoot_live_shot(src, FALSE, null, TRUE, 0) //Shitcode, but we don't have much of a choice
```

Comment:
Ditto

Reply:
Actually this one at least tries to do something, as it will prevent `null.visible_message` by using the src

Reply:
hmmmmmmm, what if it just didn't call shoot_live_shot, it's not really necessary, it mostly just applies screenshake, stamdamage and shows the message.

Reply:
Oh nevermind, it plays the sound.